### DISCUSSION TITLE
Howto create internal links to anchors within a Markdown code block

### DISCUSSION THREAD

**Discussion by DarkDau** [requarks/wiki#7064] [Help / Questions] [answered]
I am looking for a way to create internal links to anchors within a Markdown code block (~~~) in Wiki.js without disrupting the visual appearance of the content. Is there a method or escape character that can be used to break out of a code block momentarily to insert a link, similar to how \n, \t, or \b work in shell scripts? Any insights or suggestions would be greatly appreciated. Thank you

**Comment by NGPixel** [comment, marked as answer]
No
